tools/sv-report

```diff
@@ -272,177 +217,430 @@ meta_tags, meta_urls = readConfig(args.meta_tags)
+
+        test_result.timeout = int(test_log_data["timeout"])
+
+        test_result.total_time = float(test_log_data["time_elapsed"])
+        test_result.user_time = float(test_log_data["user_time"])
+        test_result.system_time = float(test_log_data["system_time"])
+
+        tool_results.total_time += test_result.total_time
+        tool_results.user_time += test_result.user_time
+        tool_results.system_time += test_result.system_time
+
+        test_result.ram_usage = float(test_log_data["ram_usage"])  # KB
+        ram_usage_mb = test_result.ram_usage / 1000
+        if tool_results.max_ram_usage < ram_usage_mb:
+            tool_results.max_ram_usage = ram_usage_mb
+
+        logs_out_dir = os.path.join(out_dir, "logs")
+        log_html = os.path.join(logs_out_dir, t_id + ".html")
+
+        test_result.log_html_file = os.path.relpath(log_html, out_dir)
+
+        # Render the log if needed
+        if not exists_and_is_newer_than(
+                log_html, [log_file, args.log_template, __file__]):
+            rendered_count += 1
+            renderLogToHTMLFile(
+                out_dir, log_html, test_result, test_log_data, log_content)
+
+        for tag in test_result.tags:
+            tag_statuses = tags_statuses.setdefault(tag, TagStatuses())
+            if test_result.status == TestStatus.PASSED:
+                tag_statuses.passed_count += 1
+            tag_statuses.present_statuses.add(test_result.status)
+            tag_statuses.present_types.update(test_result.types)
+
+    for tag, tag_statuses in tags_statuses.items():
+        tag_results = tool_results.tags[tag] = TagResults()
+
+        tag_results.passed_tests = tag_statuses.passed_count
+        if len(tag_statuses.present_statuses) == 0:
+            tag_results.status = TestStatus.NA
+        elif len(tag_statuses.present_statuses) == 1:
+            tag_results.status = list(tag_statuses.present_statuses)[0]
+            tool_results.total_tested_tags += 1
+            if (tag_results.status == TestStatus.PASSED):
+                tool_results.total_passed_tags += 1
+        else:
+            tag_results.status = TestStatus.VARIED
+            tool_results.total_tested_tags += 1
+
+        tag_results.types = list(tag_statuses.present_types)
+
+        tag_results.tests = sorted(
+            [t for t in tool_results.tests.values() if tag in t.tags],
+            key=TestResultComp)
+
+        config_js = os.path.join(
+            out_dir, "results", runner_name.lower(),
+            f"{tag.lower()}.config.js")
+
+        # Render the config
+        renderTagResultsConfig(config_js, runner_name, tag, tag_results.tests)
 
-        for tag in tags:
-            passed_count = tags[tag]["status"].count("test-passed")
-            tags[tag]["passed-num"] = passed_count
+    try:
+        with open(os.path.join(args.logs, runner_name, "version")) as f:
+            tool_results.version = f.read().strip()
+    except:
```

Comment:
Please don't leave bare except. At the very least use `except Exception`, a bare except will also catch things like ctrl+c etc which we don't want

Reply:
(here and in other places)

Reply:
Fixed